python/res/job_queue/job_queue_manager.py

```diff
@@ -148,6 +150,8 @@ def _launch_jobs(self):
                     max_submit=self.queue.max_submit
                 )
             time.sleep(1)
+            if self._queue_validator is not None:
+                self._queue_validator(self.queue, *self._queue_validator_args)
```

Comment:
When adding these lines I think the name of the function no longer reflects what the function actually does.